magento/magento2
# Catalog Rule: Update Return Type for getWebsiteIds()

### Description (*)
Updates the `getWebsiteIds()` function's return type. This function is inherited from [`Magento\Rule\Model\AbstractModel`](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Rule/Model/AbstractModel.php#L472-L484) - which defines the return type as an array, not a string.

### Manual testing scenarios (*)
1. Call the `Magento\CatalogRule\Model\Rule::getWebsiteIds()` function as part of a `foreach` loop
2. Observe the intellisense complain about an incorrect data type

![](https://user-images.githubusercontent.com/40261741/215359083-d37e2388-b6cd-466e-a110-d5b81ed941ea.png)

### Notes
Failing tests are unrelated.

### Contribution checklist (*)
